central/delegatedregistryconfig/datastore/singleton.go

```diff
@@ -0,0 +1,29 @@
+package datastore
+
+import (
+	pgStore "github.com/stackrox/rox/central/delegatedregistryconfig/store/postgres"
+	"github.com/stackrox/rox/central/globaldb"
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	once sync.Once
+
+	d DataStore
+)
+
+func initialize() {
+	if !env.PostgresDatastoreEnabled.BooleanSetting() {
```

Comment:
Probably not a question for you per se, but at what point can we stop putting these  in the code?

Reply:
I'm OK doing that now, was following what I saw elsewhere for consistency.  I see that the variable is permanently `true` so might as well remove it. For this too wasn't sure if there was a planned 'purge' of all the conditions throughout the codebase.

Reply:
Removed in latest commit